src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -115,6 +116,9 @@ public class HydrateReminderPlugin extends Plugin
 	@Inject
 	private HydrateEmojiProvider hydrateEmojiProvider;
 
+	@Inject
+	private CommandInvoker commandInvoker;
+
 //	@Inject
 //	private CommandInvoker commandDelegate;
```

Comment:
Can we just uncomment these lines and use `commandDelegate` instead of creating a new field?